KIPs/kip-97.md

```diff
@@ -0,0 +1,95 @@
+---
+kip: 97
+title: Signed Data Standard
+author: TaeRim Lee <95decode@gmail.com>
+discussions-to: https://forum.klaytn.foundation/t/kip/6359
+status: Draft
+type: Standards Track
+category : Interface
+created: 2023-01-10
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the KIP.-->
+This is to standardize message signing methods different from those of Ethereum.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+This standard proposes a specification about how to handle signed data in Klaytn.
+
+## Motivation
+<!--The motivation is critical for KIPs that want to change the Klaytn protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the KIP solves. KIP submissions without sufficient motivation may be rejected outright.-->
+Several multisignature wallet implementations have been created which accepts `presigned` transactions. A `presigned` transaction is a chunk of binary `signed_data`, along with signature (`r`, `s` and `v`). The interpretation of the `signed_data` has not been specified and ethereum solved this by EIP-191 standard. Additionally, the signature result is different in using metamask and kaikas because Klaytn does not have a standard.
```

Comment:
```suggestion
Several multisignature wallet implementations have been created which accepts `presigned` transactions. A `presigned` transaction is a chunk of binary `signed_data`, along with signature (`r`, `s` and `v`). The interpretation of the `signed_data` has not been specified in Klaytn and Ethereum solved this by [EIP-191](https://eips.ethereum.org/EIPS/eip-191) standard. Additionally, the signatures are different when using Metamask and Kaikas because they have the different preamble.
```